spec/requests/users_controller_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+  let(:user) { create(:user) }
+
+  describe 'GET #index' do
+    it 'assigns @users and renders the index template' do
+      get :index
+
+      expect(assigns(:users)).to eq(User.all)
+      expect(response).to render_template(:index)
+    end
+  end
```

Comment:
- [ ] As required for the project, kindly ensure that for each action you test all the following tests are present(test **If response status was correct**, **If a correct template was rendered**, and **If the response body includes correct placeholder text.)** as you miss some of them. Only one test has checked in this `index` action.